fs/accounting/stats.go

```diff
@@ -234,13 +241,25 @@ func (s *StatsInfo) String() string {
 	)
 
 	if !fs.Config.StatsOneLine {
+		errorFlags := make([]string, 0, 2)
+		if s.fatalError {
+			errorFlags = append(errorFlags, "fatal")
```

Comment:
I'm not sure appending "fatal" and "noretry" has enough information for the user to make sense of what is going on.

Maybe we should tell the user explicitly what is going to happen, so "Fatal error detected" or "Not retrying" (but only if there are only NoRetry errors)